pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -17,9 +17,11 @@ import (
 	ispnutil "github.com/infinispan/infinispan-operator/pkg/controller/infinispan/util"
 	ispncom "github.com/infinispan/infinispan-operator/pkg/controller/utils/common"
 	"github.com/infinispan/infinispan-operator/version"
+	openshiftv1 "github.com/openshift/api/route/v1"
```

Comment:
@rigazilla may be use routev1 for package alias?

Reply:
ok, done